package.json

```diff
@@ -78,6 +78,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.8.4",
+    "@babel/node": "^7.10.3",
```

Comment:
:fire: This upgrade is not good, also check the yarn.lock

Reply:
What upgrade? :thinking: I newly added this

Reply:
Version should be ^7.8.4